MIGRATION.md

```diff
@@ -1 +1,11 @@
-# Миграция
+# Migration
+
+## 2.1.0
+
+In the blocks `input`, `select` and `textarea` modifier `width` was taken out from the theme `islands`. It makes possible to use this modifier without theme. For the dynamic using of this modifier you should add it to the dependencies:
```

Comment:
The `width` modifier could be used independent of the theme `islands` in the `input`, `select` and `textarea` blocks. To use this modifier dynamically, add it to the dependencies: